Package/ExcelDna.AddIn/content/ExcelDna.Build.props

```diff
@@ -0,0 +1,47 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="ExcelDnaProps">
+<!--
+  If you change properties in this file, they may not come into effect until you:
+     * Rebuild the solution/project
+
+     or
+
+     * Close Visual Studio
+     * Delete .vs folder, if exists
```

Comment:
It's a known issue with some versions of Visual Studio and for some types of projects, which seem to cache the .targets (or .props) files and not automatically reload it after you make changes/save. In my tests with VS2015 and C# projects, I've never experienced this, but I did see this problem with F# projects a few months ago when I first sent the pull-request, and after some Googling, I thought it would be better to write this notice here.